### PR TITLE
Release notes for 02 April 2019

### PR DESCRIPTION
**Pull request by rhiananthony**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,170 @@
 
 <!-- newest release at the top please) -->
 
+# Prod 2019/04/02
+
+### Ingest
```
Comment (hannes-ucsc) [inline RELEASE_NOTES.md]: Nit: Is skipping level two heading intentional?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,6 +2,170 @@
...
+* Fix payload for adding file references via the ingestion service API
+
+#### Falcon v0.2.0
+* Refactor and cleaning up deployment-related files.
+* Update cromwell-tools version.
+* Update example config.json in README
+
+### Data Portal
+
+#### Azul  deployed/prod/2019-04-02__09-02 (20e526b6)
```
Comment (hannes-ucsc) [inline RELEASE_NOTES.md]: [Same objection as in the other PR](https://github.com/HumanCellAtlas/dcp/pull/323#pullrequestreview-216944314). If this is an oversight, that's fine, just fix and ignore the rest of this rant.

If you with disagree with my objection please say so. If you feel my objections are irrelevant, don't ask me for a review. That's also fine with me. Asking for my review and then ignoring my feedback is a waste of my time.

Comment (rhiananthony) [reply RELEASE_NOTES.md]: Thanks for the review. I'll fix the 2 line. Definitely not ignoring :)

File: RELEASE_NOTES.md (hunk 1 above)
Comment (rhiananthony) [reply RELEASE_NOTES.md]: Fixed. Thank you for catching that!!!

**Pull request merged**